llvm/lib/Target/RISCV/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ add_llvm_target(RISCVCodeGen
   RISCVConstantPoolValue.cpp
   RISCVDeadRegisterDefinitions.cpp
   RISCVMakeCompressible.cpp
+  RISCVLateOpt.cpp
```

Comment:
This list is supposed to be alphabetized. I don't know what happened with RISCVMakeCompressible.cpp